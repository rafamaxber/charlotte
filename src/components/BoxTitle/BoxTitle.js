import React from 'react';
import styled from 'styled-components';

const Box = styled.div`
  padding-top: 30px;
  @media (min-width: 570px) {
    padding-top: 60px;
  }
`;

const Title = styled.div`
  font-family: 'Montserrat', sans-serif;
  font-style: normal;
  font-weight: normal;
  line-height: normal;
  text-align: center;
  letter-spacing: 2.5px;
  color: #000000;
  display: block;
  font-size: 17px;
  padding: 0 20px;
  @media (min-width: 570px) {
    padding: 0 0 0 0;
    font-size: 22px;
  }
`;

const Message = Title.extend`
  font-family: 'Montserrat', sans-serif;
  font-size: 18px;
`;

export default ({ children }) => (
  <Box>
    <Title>
      { children }
    </Title>
  </Box>
)

export const BoxMessage = ({ children }) => (
  <Box>
    <Message>
      { children }
    </Message>
  </Box>
)
