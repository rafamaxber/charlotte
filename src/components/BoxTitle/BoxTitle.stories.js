import React from 'react';
import { storiesOf } from '@storybook/react';
import BoxTitle from './BoxTitle';

storiesOf('BoxTitle', module)
  .add('BoxTitle', () => (
    <BoxTitle>
      Select the dates to stay in Charlotte
    </BoxTitle>
  ));
