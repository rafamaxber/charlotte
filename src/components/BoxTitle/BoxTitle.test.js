import React from 'react';
import renderer from 'react-test-renderer';
import BoxTitle from './BoxTitle';

describe('BoxTitle', () => {
  it('Should match with snapshot', () => {
    const component = renderer.create(
      <BoxTitle></BoxTitle>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
