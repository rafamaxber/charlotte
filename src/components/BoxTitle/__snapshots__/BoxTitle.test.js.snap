// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxTitle Should match with snapshot 1`] = `
<div
  className="sc-bdVaJa cnOyFf"
>
  <div
    className="sc-bwzfXH fccHGK"
  />
</div>
`;
